server/framework/standalone.lua

```diff
@@ -0,0 +1,9 @@
+--- @param playerId number
+--- @return any
+Framework.getPlayer = function(playerId)
+    if not playerId then return nil end
+
+    return {
+
+    }
```

Comment:
Shouldn't this return the player and not an empty table?

Reply:
I should define some some placeholder data, will see.